pkg/sdk/action.go

```diff
@@ -54,6 +56,43 @@ func Create(object Object) (err error) {
 	return nil
 }
 
+// Patches provided object on the server with given patch and updates the arg
+// "object" with the result from the server(UID, resourceVersion, etc).
+// Returns an error if the object’s TypeMeta(Kind, APIVersion) or ObjectMeta(Name, Namespace) is missing or incorrect.
+// Returns an error if patch couldn't be json serialized into bytes.
+// Can also return an api error from the server
+// e.g Conflict https://github.com/kubernetes/apimachinery/blob/master/pkg/api/errors/errors.go#L428
+func Patch(patch interface{}, object Object, pt types.PatchType) (err error) {
+	name, namespace, err := k8sutil.GetNameAndNamespace(object)
+	if err != nil {
+		return err
+	}
+	gvk := object.GetObjectKind().GroupVersionKind()
```

Comment:
line between 69 to 70?

Reply:
you mean add an empty new line after this ?

btw, I copied same structure here as in other actions e.g. Create.

Reply:
yes, it is just a style nits. and maybe we forget to do that in Create.

Reply:
ok, changed in all places in action.go , btw I see similar style differences in `query.go` as well... haven't touched that . more of nits I guess.